api-v3-beta.rst

```diff
@@ -0,0 +1,62 @@
+.. post:: August 8, 2019
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+
+      Announcing API v3 Beta as invite-only status. Help us to make it *even* better!
+
+========================
+ Announcing API v3 Beta
+========================
+
+In the last months, we have been working on making our API better.
+Considering the limitations of our current REST API v2,
+we decided to make a bigger step forward and create our new shiny API v3 with better and bigger features,
+putting the focus on the user cases we heard about from existing users.
+
+Our new API v3 has some big differences compared to the old one that make it more user-friendly and useful.
+
+These differences includes:
+
+* Authenticated only
+* Token based authentication
+* Read and Write
+* Project's slug resource based
+
+Allowing us to build new good features on top of them, like:
+
+* `Import a new project`_
+* `List all my projects`_
+* `Activate a version`_
+* `Trigger a build for a specific version`_
+* `Check build status`_
+* `Manage redirects`_ and `environment variables programmatically`_
+
+.. _Import a new project: https://docs.readthedocs.io/page/api/v3.html#project-create
+.. _List all my projects: https://docs.readthedocs.io/page/api/v3.html#projects-list
+.. _Activate a version: https://docs.readthedocs.io/page/api/v3.html#version-update
+.. _Trigger a build for a specific version: https://docs.readthedocs.io/page/api/v3.html#build-triggering
+.. _Check build status: https://docs.readthedocs.io/page/api/v3.html#build-details
+.. _Manage redirects: https://docs.readthedocs.io/page/api/v3.html#redirects
+.. _environment variables programmatically: https://docs.readthedocs.io/page/api/v3.html#environment-variables
+
+If you want to know more about it,
+please check out the full `APIv3 documentation`_.
+
+.. _APIv3 documentation: https://docs.readthedocs.io/page/api/v3.html
+
+
+Invite-only Beta feature
```

Comment:
Mixed case here. I prefer lowercase for headings ("Invite-only beta feature") over title case ("Invite-only Beta Feature"), but either is fine -- consistency with headings is more important.

Reply:
`Invite-only beta feature` is the way to go to keep consistency with other posts.